Makefile

```
@@ -16,6 +16,9 @@ test: package_vulnerability flake unit_tests component_tests
 unit_tests:
 	pipenv run pytest test/unit/
 
+unit_test_coverage:
+	pipenv run pytest test/unit/ --cov app --cov-report html
```

```suggestion
	pipenv run pytest test/unit/ --cov app --cov-report html --cov-report term-missing
```

Gives a nice little command line summary of the coverage as well.